src/colorMap.js

```diff
@@ -7,6 +7,7 @@ module.exports = {
   '#EBF1F3': '--color-accent-g3',
   '#7E94A4': '--color-accessory',
   '#C1D4DB': '--color-accessory-light',
+  '#BBDEE8': '--color-background-active',
```

Comment:
not sure exactly what your intended usage of this is, but it's likely you'll need to add this color to `colors.css` directly vs. putting it here. I _think_ this file is basically dead/unused